deb/openmediavault/usr/share/php/openmediavault/system/storage/smartinformation.inc

```diff
@@ -252,6 +252,13 @@ class SmartInformation {
 			$attrData['assessment'] = self::SMART_ASSESSMENT_BAD_ATTRIBUTE_IN_THE_PAST;
 			return;
 		}
+		// mark bad sector attributes as red when rawvalue >= 1, ticket #1772
```

Comment:
Please use this comment.

// Check for bad sectors (Reallocated_Sector_Ct and Current_Pending_Sector).